src/Facebook/Facebook.php

```diff
@@ -576,7 +576,7 @@ public function videoToUpload($pathToFile)
     /**
      * Upload a video in chunks.
      *
-     * @param int $target The id of the target node before the /videos edge.
+     * @param string $target The id of the target node before the /videos edge.
```

Comment:
Since these changes are targeting the `5.x` branch, let's go with union type syntax for backward compatibility. So in this case, it'd be `int|string`. :)